### PR TITLE
Add comments

### PR DESCRIPTION
**Pull request by Ruruthia**
This pr adds comments actions: new, create, index.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -53,6 +53,7 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'pry'
```
Comment (korneliacieslik) [inline Gemfile]: What pray does?

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    if @comment.save
+      redirect_to idea_comments_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+
+  end
```
Comment (korneliacieslik) [inline app/controllers/comments_controller.rb]:
```ruby
def edit
end 
```

it looks better :)

File: app/controllers/comments_controller.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(:user_name, :body)
+  end
+
+  def idea
+    @idea ||= Idea.find(params[:idea_id])
```
Comment (korneliacieslik) [inline app/controllers/comments_controller.rb]: Maybe it makes more sense? 
``` ruby
@idea ||= current_user.ideas.find(params[:idea_id])
```

File: app/models/comment.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :idea
+  validates :user_name, :body, presence: true
+  scope :ordered_by_newest, -> { order('created_at DESC NULLS LAST')}
```
Comment (korneliacieslik) [inline app/models/comment.rb]: `NULLS LAST` is it necessary?

**Review by korneliacieslik — COMMENTED** [review]
Please review my comments

File: Gemfile (hunk 1 above)
Comment (Ruruthia) [reply Gemfile:56]: You can use this gem to debug.

File: app/models/comment.rb (hunk 4 above)
Comment (Ruruthia) [reply app/models/comment.rb:4]: Yep, because we want to have comments with 'created_at: NULL' appear at the bottom of comments view.

**Pull request closed without merging**